solid/src/main/java/com/inrupt/client/solid/SolidResourceReference.java

```diff
@@ -49,6 +53,7 @@ public SolidResourceReference(final URI identifier, final Metadata metadata) {
             this.metadata = metadata;
         }
         this.entity = new ByteArrayInputStream(new byte[0]);
+        this.headers = Headers.of(new HashMap<>());
```

Comment:
`Headers.of(new HashMap<>())` is used so many times, maybe it's worth hiding behind a new `Headers.of()` (like `Set.of()` etc.) or `Headers.empty()`.

Thought `Set.of()` and the like tend to be immutable, so maybe a different name.

Reply:
Good idea. I added a `Headers.empty()` method in https://github.com/inrupt/solid-client-java/pull/774/commits/5927dbe302a42a60dfa7d2424e2ea69504d189a5